pyproject.toml

```
@@ -3,7 +3,7 @@ name = "nitypes"
 description = "Data types for NI Python APIs"
 license = "MIT"
 keywords = ["nitypes"]
-version = "1.1.0.dev1"
+version = "1.2.0.dev0"
```

We already bumped minor when we released 1.0.0. We haven't released 1.1.0 yet, so don't bump it again.
```suggestion
version = "1.1.0.dev1"
```